doc/Configuration-Options.md

```diff
@@ -0,0 +1,227 @@
+# Configuring OmniSharp
```

Comment:
I would only link from here to the wiki, otherwise we have to maintain the docs in two places.
and Wiki is preferable cause anyone can edit at any point, without a PR

Reply:
Yeah, I thought so.
It's just been updated.